build/publish-on-homebrew.sh

```diff
@@ -34,6 +34,8 @@ darwinArmZipUrl="https://downloads.apache.org/maven/mvnd/${VERSION}/maven-mvnd-$
 darwinArmSha256="$(curl -L --silent "${darwinArmZipUrl}.sha256")"
 linuxZipUrl="https://downloads.apache.org/maven/mvnd/${VERSION}/maven-mvnd-${VERSION}-linux-amd64.zip"
```

Comment:
maybe add the `Amd` prefix to keep things aligned ? similar to what we have for `darwin`

Reply:
I have fixed that. Please review again :-)